library-best-practices/JLBP-9.md

```diff
@@ -12,3 +12,9 @@ consumers advance their minimum version of Java before they advance it
 themselves. (Waiting for every last consumer could block an upgrade permanently,
 due to cases like abandoned projects.)
 
+As an alternative, if there is sufficient need for supporting both consumers on
+the old version of Java and consumers wanting features provided by a newer
+version of Java, a long-term support (LTS) release can be created to continue
```

Comment:
not sure if we want to use the phrase "long-term support" here. The main point is that you can release new minor and micro versions while also releasing new major versions, but we can futz with the wording later. The intent is fine.

Reply:
Ok, I'll keep it for now because I don't have an alternative yet.